server/app/services/geo/esri/EsriServiceAreaValidationOption.java

```diff
@@ -0,0 +1,36 @@
+package services.geo.esri;
+
+import com.google.auto.value.AutoValue;
+
+/** Represents a service area to validate against. */
+@AutoValue
+public abstract class EsriServiceAreaValidationOption {
+  public static Builder builder() {
+    return new AutoValue_EsriServiceAreaValidationOption.Builder();
+  }
+
+  /** Returns a human readable label used for display in the UI. */
+  public abstract String getLabel();
+
+  /** Returns a value. Used in conjunction with path for service area validation. */
+  public abstract String getValue();
+
+  /** Returns a URL to call an external Esri boundary layer service. */
+  public abstract String getUrl();
+
+  /** Returns a JsonPath. Used in conjunction with value for service area validation. */
+  public abstract String getPath();
```

Comment:
**suggestion (non-blocking):** if this is the correct doc page please link to it here: https://support.esri.com/en-us/gis-dictionary/path